src/Functional.CQS.AOP.CommonTestInfrastructure/DummyObjects/DummyAsyncCommandHandlerThatFails.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Reflection;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Functional.CQS.AOP.CommonTestInfrastructure.DummyObjects
+{
+	/// <summary>
+	/// Sample <see cref="IAsyncCommandHandler{TCommand, TError}"/> implementation.  Returns a failure result.
+	/// </summary>
+	public class DummyAsyncCommandHandlerThatFails : IAsyncCommandHandler<DummyAsyncCommandThatFails, DummyAsyncCommandError>
+	{
+		private static readonly DummyAsyncCommandThatFails _command = new DummyAsyncCommandThatFails();
+		private static readonly Result<Unit, DummyAsyncCommandError> _result = Result.Failure<Unit, DummyAsyncCommandError>(new DummyAsyncCommandError());
+
+		/// <summary>
+		/// Handles the command asynchronously.
+		/// </summary>
+		/// <param name="command">The command parameters.</param>
+		/// <param name="cancellationToken">The cancellation token.</param>
+		/// <returns></returns>
```

Comment:
Does `<returns>A failure result.</returns>` make sense here?

Reply:
Similar comment goes for `DummyAsyncCommandHandlerThatSucceeds`, with `success` instead of `failure`.

Reply:
Similar comments go for the other dummy handlers.

Reply:
I tend to not fill out the `returns` tag as the function name + return type should be self-documenting.  I think it only actually matters if I'm generating a documentation page?  AFAIK, `returns` information does not show up in Intellisense.

Reply:
No, I don't believe it does. I think it's only used by any documentation generator.